scipy/spatial/distance.py

```diff
@@ -1421,10 +1481,6 @@ def dfun(u, v):
         elif mstr in ['minkowski', 'mi', 'm']:
             X = _convert_to_double(X)
             _distance_wrap.pdist_minkowski_wrap(X, dm, p)
-        elif mstr in ['wminkowski', 'wmi', 'wm', 'wpnorm']:
-            X = _convert_to_double(X)
-            w = _convert_to_double(np.asarray(w))
-            _distance_wrap.pdist_weighted_minkowski_wrap(X, dm, p, w)
```

Comment:
Actually, these lines of code should never get called anyway, since if `w` is not `None`, mstr is changed to ensure it starts with `test_`. So we could leave this code in, but this branch would never be traversed unless someone called wminkowski with no weights.